frontend/dashboard/pages/PageLayout/DashboardHeader/DashboardHeader.tsx

```diff
@@ -48,20 +50,32 @@ function TopNavigationMenuItem({ menuItem }: TopNavigationMenuProps): React.Reac
   const selectedContext: string = useSelectedContext();
   const { t } = useTranslation();
   const path: string = `${menuItem.link}/${selectedContext}`;
+  const currentRoutePath: string = extractSecondLastRouterParam(location.pathname);
 
   return (
     <StudioPageHeader.HeaderLink
       color='dark'
       variant='regular'
       renderLink={(props) => (
         <NavLink to={path} {...props}>
-          {t(menuItem.name)}
+          <span
+            className={cn({
+              [classes.active]: menuItem.key === currentRoutePath,
```

Comment:
God idé med å bruke `key`-en til menyelementet, men jeg har et forslag til:
```suggestion
              [classes.active]: StringUtils.removeLeadingSlash(menuItem.link) === currentRoutePath,
```
Hvis vi gjør det slik, vil begge sider av likhetstegnet være koblet til konstantene i `constants.js`. Da slipper vi å huske å manuelt oppdatere `HeaderMenuItemKey`, hvis stiene endres en gang i fremtiden 🙂

Reply:
Genialt! 🤩